docs/source/guides/build.rst

```diff
@@ -0,0 +1,218 @@
+
+Building Iroha
+==============
+
+In this guide we will learn how to install all dependencies, required to build 
+Iroha and how to build it.
+
+Preparing the environment
+-------------------------
+
+In order to successfully build Iroha, we need to configure the environment. 
+There are several ways to do it and we will describe all of them.
+
+Currently we support Unix-like systems (we are basically targeting popular 
+Linux distros and macOS). If you happen to have Windows or you don't want to 
+spend time installing all dependencies, you might want to consider using Docker
+environment.
+
+.. hint:: Having troubles? Check FAQ section or communicate to us directly, in
+  case you were stuck on something. We don't expect this to happen, but some
+  issues with an environment are possible.
+
+Docker
+^^^^^^
+First of all, you need to install ``docker`` and ``docker-compose``. You can 
+read how to install it on a 
+`Docker's website <https://www.docker.com/community-edition/>`_
+
+.. note:: Please, use the latest available docker daemon and docker-compose. 
+  If you have troubles launching the Docker container — it is likely that those 
+  problems are caused by an outdated version.
+ 
+After it clone the `Iroha repository <https://github.com/hyperledger/iroha>`_ 
+to the directory of your choice.
+
+.. code-block:: shell
+
+  git clone -b develop https://github.com/hyperledger/iroha
+
+After it, you need to run the development environment. Run the script 
+``run-iroha-dev.sh``, contained in the folder ``scripts``: 
+
+.. code-block:: shell
+
+  sh ./iroha/scripts/run-iroha-dev.sh
```

Comment:
It is better to remove iroha folder from the path because the user could issue git clone with a custom folder instead of default iroha.

Reply:
Please remove sh or write **bash** instead, because the script has to run in bash. It would not work in sh.